### ISSUE TITLE
When backspace is long-pressed only one character is deleted

### ISSUE THREAD

**Issue by a-ch** [egslava/edittext-mask#11]

**Comment by egslava** [comment]
Hi, @a-ch ! I'm preparing to a job interview, so don't have a lot of time next 7 days. Have you figured out what's going on?

**Comment by a-ch** [comment]
I guess this is something about managing the cursor's position, this might prevent next character from being deleted when backspace is long-pressed.

**Comment by egslava** [comment]
Hi, @a-ch!
Is it still actual or have you solved the issue? :)

**Comment by a-ch** [comment]
Nope, had no time to look into it

**Comment by arturwest** [comment]
@egslava still no updates for this issue?

**Comment by egslava** [comment]
@arturwest hey :)
I'm still looking for a job in another country, so my visa is expired soon and I need to seek more actively :) So, yep, _from me_ still no updates :)

But I rely on you, guys :) I'm pretty sure it's a fixable error for one-two nights. Perhaps, are you tough enough for it? :)

**Comment by yccheok** [comment]
@egslava 

Hi, hows your job searching? Any plan to solve this issues?

**Comment by MahdiAstanei** [comment]
fix it please

**Comment by egslava** [comment]
Agree, the behavior is pretty annoying. A pull-request is very welcome, I'll provide all the info support I can provide in case if you want to fix it. Please, don't hesitate to contact or, seems, I'll fix it 'a bit' later myself :)
